templates/cli/lib/parser.js.twig

```diff
@@ -203,6 +203,7 @@ const commandDescriptions = {
     "databases": `The databases command allows you to create structured collections of documents, query and filter lists of documents.`,
     "init": `The init command provides a convenient wrapper for creating and initializing project, functions, collections, buckets, teams and messaging in Appwrite.`,
     "push": `The push command provides a convenient wrapper for pushing your functions, collections, buckets, teams and messaging.`,
+    "run": `The dev command allows you to run project locally to allow easy development and quick debugging.`,
```

Comment:
```suggestion
    "run": `The run command allows you to run project locally to allow easy development and quick debugging.`,
```
Also we need to discuss if this needs to be `run` or `dev` as we need to keep some future products in mind as well